tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/getstarted-welcome.cy.js

```diff
@@ -0,0 +1,36 @@
+// <reference types="Cypress" />
+
+describe('Get Started Welcome Page', function () {
+
+    before(() => {
+        cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/welcome');
+        cy.injectAxe(); 
+        cy.wait(5000);
+    });
+
+    it('Check if Drawer is closed', () => {
+        cy.get('.nfd-onboarding-drawer__panel-inner').should('not.be.visible');
+    });
+
+    it('Check if welcome card is visible', () => {
+        cy.get('.welcome-card').should('be.visible');
+    });
+
+    it('Check if Header and Subheader shows up', () => {
+        cy.get('.nfd-step-card-heading').should('be.visible');
+        cy.get('.nfd-step-card-subheading').should('be.visible');
+    });
+
+    it('Change tab and check', () => {
+        cy.get('#tab-panel-0-tab2').click().should('be.visible');
+        cy.get('#tab-panel-0-tab2-view').should('be.visible').and('contain', 'POWERFUL FEATURES');
```

Comment:
One last thing I missed that time is that we should not be doing `contain` checks because this can easily fail if the text gets translated to another languages

Reply:
I have removed the ".and('contain', 'POWERFUL FEATURES');" part from the test, since the only way to check for presence of text is contain or include and in that too we need to provide the text to test. The test still checks whether the second div is visible so that should work across languages.